### DISCUSSION TITLE
Doesn't Scan the whole text file

### DISCUSSION THREAD

**Discussion by deeydos** [projectdiscovery/httpx#1191] [Q&A]
i had 400 subdomains in a text file i filtered it with httpx -sc -title > newdomains.txt and stored them in new text file
when i checked the length of newdomains.txt with wc -l newdomains.txt
they were only 197 
and it missed more than half of subdomains why is that

**Comment by MetzinAround** [comment]
Do you have any more information about what happened? knowing what both wordlists look like might help. 

Did the output have a variety of codes or only 200s?

**Comment by ehsandeep** [comment]
@deeydos number of subdomains is not the same as a number of web servers, httpx will only display subdomains with active web service, so the results got filtered.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
